src/handlers/payout/post.ts

```diff
@@ -398,7 +398,7 @@ const calculateRewardValue = (comments: Record<string, string[]>, incentives: In
       const reward = wordReward.mul(value.map((str) => str.trim().split(" ").length).reduce((totalWords, wordCount) => totalWords + wordCount, 0));
       sum = sum.add(reward);
     } else {
-      if (!incentives.comment.elements[key]) {
+      if (!incentives.comment.elements[key] || incentives.comment.ignore_children.includes(key)) {
```

Comment:
this is not desired functionality. this will only stop processing that particular item, but we want stop recursing when we encounter that item during parsing the markdown. 
it basically replaces hardcoded variable: https://github.com/ubiquity/ubiquibot/blob/dba8308779aa1ea74e5a98c227a5e92ae3307d9b/src/handlers/payout/post.ts#L18

Reply:
> this is not desired functionality. this will only stop processing that particular item, but we want stop recursing when we encounter that item during parsing the markdown. it basically replaces hardcoded variable:
> 
> https://github.com/ubiquity/ubiquibot/blob/dba8308779aa1ea74e5a98c227a5e92ae3307d9b/src/handlers/payout/post.ts#L18

ohhh my bad

Reply:
> it basically replaces hardcoded variable:

Shouldn't we remove the hardcoded variable if we can handle from the config @whilefoo? If so, would you mind creating a new issue for that?

Reply:
> > it basically replaces hardcoded variable:
> 
> Shouldn't we remove the hardcoded variable if we can handle from the config @whilefoo? If so, would you mind creating a new issue for that?

should i remove it? will replace it with incentive ignore children

Reply:
I think it makes more sense to handle in a new issue.

Reply:
> I think it makes more sense to handle in a new issue if whilefoo thinks its necessary. That way this one can be merged in faster, most likely.

sure

Reply:
> Shouldn't we remove the hardcoded variable if we can handle from the config @whilefoo? If so, would you mind creating a new issue for that?

This issue is about this hardcoded variable, it stops recursing when it encounters one of the tags in this variable. So we just need to move it the config as `ignore-children`